### PR TITLE
[FrameworkBundle][Workflow] Clean code

### PR DESCRIPTION
**Pull request by lyrixx**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       |
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd [hunk 1]
```diff
@@ -398,8 +398,6 @@
 
     <xsd:simpleType name="marking_store_type">
         <xsd:restriction base="xsd:string">
-            <xsd:enumeration value="multiple_state" />
-            <xsd:enumeration value="single_state" />
```
Comment (lyrixx) [inline src/Symfony/Bundle/FrameworkBundle/Resources/config/schema/symfony-1.0.xsd:402]: We have remove this support a long time ago (cbb8b22753)

File: src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/workflows.xml [hunk 2]
```diff
@@ -10,7 +10,6 @@
         <framework:workflow name="article" type="workflow">
             <framework:audit-trail enabled="true"/>
             <framework:initial-marking>draft</framework:initial-marking>
-            <framework:marking-store type="method" property="state" />
```
Comment (lyrixx) [inline src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/xml/workflows.xml:13]: This is already the default value. I removed it to be iso with PHP, and YAML

File: src/Symfony/Component/Workflow/Tests/MarkingStore/MethodMarkingStoreTest.php [hunk 3]
```diff
@@ -26,7 +25,6 @@ public function testGetSetMarkingWithMultipleState()
 
         $marking = $markingStore->getMarking($subject);
 
-        $this->assertInstanceOf(Marking::class, $marking);
```
Comment (lyrixx) [inline src/Symfony/Component/Workflow/Tests/MarkingStore/MethodMarkingStoreTest.php:29]: This is useless, because enforced by the interface typing

**Comment by nicolas-grekas** [comment]
Thank you @lyrixx.

**Pull request merged**